dev/Scroller/Scroller.cpp

```diff
@@ -808,10 +808,11 @@ winrt::Size Scroller::ArrangeOverride(winrt::Size const& finalSize)
         viewport.Width          /*viewportWidth*/,
         viewport.Height         /*viewportHeight*/);
 
-#ifndef USE_EFFECTIVE_VIEWPORT_AND_ANCHORING_FROM_PLATFORM
-    ClearAnchorCandidates();
-    RaisePostArrange();
-#endif
+    if (!SharedHelpers::IsRS5OrHigher())
```

Comment:
It's not obvious from these blocks _why_ they need to be making version checks

Reply:
Will add some comments.